### PR TITLE
Add instructions for Scylla

### PR DESCRIPTION
**Pull request by ashao**

### CODE REVIEW

**Review by MattToast — APPROVED** [review]
LGTM!!

File: doc/installation_instructions/platform/pml-scylla.rst [hunk 1]
```diff
@@ -0,0 +1,75 @@
+PML Scylla
+==========
+
+Notes
+-----
+
+As of September 2024, the software stack on Scylla is still being
```
Comment (amandarichardsonn) [inline doc/installation_instructions/platform/pml-scylla.rst]: What about: As of September 2024, the software stack on Scylla is still under development. Therefore, please consider these instructions as preliminary for now.

File: doc/installation_instructions/platform/pml-scylla.rst [hunk 2]
```diff
@@ -0,0 +1,75 @@
+PML Scylla
+==========
+
+Notes
```
Comment (amandarichardsonn) [inline doc/installation_instructions/platform/pml-scylla.rst]: I might instead make this a warning -> https://sublime-and-sphinx-guide.readthedocs.io/en/latest/notes_warnings.html#:~:text=Warnings%20are%20formatted%20in%20the,..

File: doc/installation_instructions/platform/pml-scylla.rst [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+    [SmartSim] INFO Verifying TensorFlow Backend
+    16:26:35 login SmartSim[557020:MainThread] INFO Success!
+
+Post-installation
+-----------------
+
+After completing the above steps to install SmartSim in a conda environment, you
+can reload the conda environment by running the following commands:
+
+.. code:: bash
```
Comment (amandarichardsonn) [inline doc/installation_instructions/platform/pml-scylla.rst:80]: suspect comment! `# # cudnn should NOT be loadded`

File: doc/installation_instructions/platform/pml-scylla.rst [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+
+    pip install git+https://github.com/CrayLabs/SmartSim.git
+
+**Step 4:** Build Redis, RedisAI, the backends, and all the Python packages:
+
+.. code:: bash
+
+    export TORCH_CUDA_ARCH_LIST="8.0 8.6 8.9 9.0" # Workaround for a PyTorch problem
```
Comment (amandarichardsonn) [inline doc/installation_instructions/platform/pml-scylla.rst:42]: Do you think any more information should be give for the comment `# Workaround for a PyTorch problem` - maybe in the form of a note -> https://sublime-and-sphinx-guide.readthedocs.io/en/latest/notes_warnings.html#notes

Im wondering if user will think itll affect their work? Mayhaps not

**Review by amandarichardsonn — COMMENTED** [review]
Splendid work, just a few minor enchantments suggested!

File: doc/installation_instructions/platform/pml-scylla.rst (hunk 2 above)
Comment (amandarichardsonn) [reply doc/installation_instructions/platform/pml-scylla.rst]: the Scylla info ** also is it kinda fingers crossed it works right now? Or are you mentioning the under development bc of the bugs commented below in the instructions? the pytorch and other issue

File: doc/installation_instructions/platform/pml-scylla.rst (hunk 3 above)
Comment (ashao) [reply doc/installation_instructions/platform/pml-scylla.rst:80]: This comment is true, we do not want people loading cudnn

File: doc/installation_instructions/platform/pml-scylla.rst (hunk 4 above)
Comment (ashao) [reply doc/installation_instructions/platform/pml-scylla.rst:42]: This is a very preliminary thing for now and it's likely that things will change. The workshop I'm giving tomorrow is basically a "sneak peak" kinda thing. So it should be ok for now. That being said, I agree that I should capture why we have to do this in a note

Comment (amandarichardsonn) [reply doc/installation_instructions/platform/pml-scylla.rst:42]: I see!

File: doc/installation_instructions/platform/pml-scylla.rst [hunk 5]
```diff
@@ -0,0 +1,85 @@
+PML Scylla
+==========
+
+.. warning::
+    What about: As of September 2024, the software stack on Scylla is still
```
Comment (amandarichardsonn) [inline doc/installation_instructions/platform/pml-scylla.rst]: I dont think you need -> "What about: "

File: doc/installation_instructions/platform/pml-scylla.rst [hunk 6]
```diff
@@ -0,0 +1,85 @@
...
+
+Post-installation
+-----------------
+
+After completing the above steps to install SmartSim in a conda environment, you
+can reload the conda environment by running the following commands:
+
+.. code:: bash
+
+    module load cudatoolkit/12.4.1 git # # cudnn should NOT be loadded
```
Comment (amandarichardsonn) [inline doc/installation_instructions/platform/pml-scylla.rst]: Looks like the double # # and the loadded misspelled still exists!

File: doc/installation_instructions/platform/pml-scylla.rst [hunk 7]
```diff
@@ -0,0 +1,85 @@
...
+    export TORCH_CUDA_ARCH_LIST="8.0 8.6 8.9 9.0" # Workaround for a PyTorch problem
+    smart build --device=cuda-12
+    module unload cudnn # Workaround for a PyTorch problem
+
+
+.. note::
+    The first workaround is needed because for some reason the autodetection
+    of CUDA architectures is not consistent internally with one of PyTorch's
+    dependencies. This seems to be unique to this machine as we do not see
+    this on other platforms
```
Comment (amandarichardsonn) [inline doc/installation_instructions/platform/pml-scylla.rst]: Add . to complete sentence

File: doc/installation_instructions/platform/pml-scylla.rst [hunk 8]
```diff
@@ -0,0 +1,85 @@
...
+
+    The second workaround is needed because PyTorch 2.3 (and possibly 2.2)
+    will attempt to load the version of cuDNN that is in the LD_LIBRARY_PATH
+    instead of the version shipped with PyTorch itself. This results in
+    unfound symbols
```
Comment (amandarichardsonn) [inline doc/installation_instructions/platform/pml-scylla.rst]: Add . to complete sentence

**Review by amandarichardsonn — APPROVED** [review]
LGTM! Thanks for making those changes

File: doc/installation_instructions/platform/pml-scylla.rst (hunk 3 above)
Comment (amandarichardsonn) [reply doc/installation_instructions/platform/pml-scylla.rst:80]: I think loaded is misspelled tho! And theres two # !

**Pull request merged**